### PR TITLE
[CV-423][Web] Annual Financial Report Generation

### PR DESCRIPTION
**Pull request by Kadestery**
**Click up Issue:** https://app.clickup.com/t/86azdbnvm

### Description:
PR implements the 'Finance' page. Integrates the PropertyFinanceCard into the FinanceComponent. Allows for Condo Management Companies to view financial reports of their properties.

#### User flow:
-CMC clicks on 'Finance' in navbar/dashboard
-CMC clicks on a property, finances of property is displayed

### Demo:
AnnualFinancialReport
Pr test 2
Pr test 3

### CODE REVIEW

**Comment by Kadestery** [comment]
End-to-End tests passed prior to the refactoring push, working on more refactoring tomorrow once firebase limit resets

**Review by Fetyali7 — APPROVED** [review]
It would be great if you added more comments explaining things but other than that, it looks good to me, nicely done Kade!

**Review by XavierGuertin — APPROVED** [review]

**Review by XavierGuertin — CHANGES_REQUESTED** [review]
add some comments at the top of the class explaining what it does

File: apps/native/screens/PropertyManagementScreen.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ import { db } from "../firebase";
 import { getDocs, collection, query, where } from "firebase/firestore";
 import AsyncStorage from "@react-native-async-storage/async-storage";
 import PropertyAdapter from "@native/components/PropertyAdapter";
-import CondoUnitAdapter from "@native/components/CondoUnitAdapter";
+import { processCondoUnits } from '../../web/app/processCondoUnits';
```
Comment (Kadestery) [inline apps/native/screens/PropertyManagementScreen.tsx:19]: sa

**Review by Kadestery — COMMENTED** [review]
Added comments in new push

**Review by Fetyali7 — APPROVED** [review]
Looks better now Kade!

**Review by XavierGuertin — APPROVED** [review]

**Review by Adissuu — APPROVED** [review]
Good job, good following of the code convention

**Pull request merged**